### ISSUE TITLE
Compiling src on ARM arch Linux fails at properties-cpp not found

### ISSUE THREAD

**Issue by RubyRebbe** [anbox/anbox#1500]
Cloned anbox GitHub repo to Pinebook 64, running Manjaro ARM arch Linux. Cmake runs along happily until fails at:  properties-cpp not found. 
Any ideas? Thanks!
-- RubyRebbe

**Comment by necessarily-equal** [comment]
You should install properties-cpp before building. It's one of the Anbox dependencies

**Comment by RubyRebbe** [comment]
Necessarily-equal, thanks for responding.
I am running arch linux on ARM, not X86. When I run the package installer 
pacman -S properties-cpp

it responds:
error:  target not found:  properties-cpp

I tried installing the source from Xenial Ubuntu repo, that succeeded.
They are just cpp header files, no cpp source files,  no binaries, located in /usr/include/core:

connection.h, property.h, signal.h

cmake was still unable to see that I had installed the package.
I suspect that there is some cmake magic that I do not understand, and that I need to set up.

Any ideas?
Thanks!
